diff --git a/configs/all.js b/fixed/all.js
index c087aa8..d45c49b 100644
--- a/configs/all.js
+++ b/fixed/all.js
@@ -3,7 +3,7 @@
 // DO NOT EDIT THIS CODE BY HAND
 // YOU CAN REGENERATE IT USING yarn generate:configs
 module.exports = {
-    extends: ['./configs/base', './configs/eslint-recommended'],
+    extends: ['plugin:@typescript-eslint/base', './configs/eslint-recommended'],
     rules: {
         '@typescript-eslint/adjacent-overload-signatures': 'error',
         '@typescript-eslint/array-type': 'error',
@@ -169,4 +169,4 @@ module.exports = {
         '@typescript-eslint/unified-signatures': 'error',
     },
 };
-//# sourceMappingURL=all.js.map
\ No newline at end of file
+//# sourceMappingURL=all.js.map
diff --git a/configs/recommended-requiring-type-checking.js b/fixed/recommended-requiring-type-checking.js
index d53a3e1..0672433 100644
--- a/configs/recommended-requiring-type-checking.js
+++ b/fixed/recommended-requiring-type-checking.js
@@ -24,4 +24,4 @@ module.exports = {
         '@typescript-eslint/unbound-method': 'error',
     },
 };
-//# sourceMappingURL=recommended-requiring-type-checking.js.map
\ No newline at end of file
+//# sourceMappingURL=recommended-requiring-type-checking.js.map
diff --git a/configs/recommended.js b/fixed/recommended.js
index 251eff6..b681850 100644
--- a/configs/recommended.js
+++ b/fixed/recommended.js
@@ -34,4 +34,4 @@ module.exports = {
         '@typescript-eslint/triple-slash-reference': 'error',
     },
 };
-//# sourceMappingURL=recommended.js.map
\ No newline at end of file
+//# sourceMappingURL=recommended.js.map
diff --git a/configs/strict.js b/fixed/strict.js
index 3be056d..c79efe2 100644
--- a/configs/strict.js
+++ b/fixed/strict.js
@@ -47,4 +47,4 @@ module.exports = {
         '@typescript-eslint/unified-signatures': 'warn',
     },
 };
-//# sourceMappingURL=strict.js.map
\ No newline at end of file
+//# sourceMappingURL=strict.js.map
